docs/insights-archive-sample/config/persistentvolumes/pvc-{hash-id}.json

```diff
@@ -0,0 +1,70 @@
+{
+    "metadata": {
+        "name": "pvc-99ffaeb3-8ff8-4137-a1fc-0bf72e7209a5",
```

Comment:
It's a PV not PVC, right? I would be rather explicit in the sample archive examples.

Reply:
This is a copy from a PV I gathered from a cluster, the id apparently references the PVC who bound it.
I'll edit the sample to avoid errors, thanks.

Reply:
Ahh OK....interesting.Thanks